mirrormanager2/app.py

```diff
@@ -373,14 +375,22 @@ def host_new(site_id):
 
         try:
             SESSION.flush()
+            status = 'Host added'
             flask.flash('Host added')
         except SQLAlchemyError as err:
             SESSION.rollback()
+            status = 'Could not create the new host'
```

Comment:
@ralphbean @pypingou removing the status messages to be sent when it fails to do an action.